bld/CLMBuildNamelist.pm

```diff
@@ -3705,7 +3705,7 @@ sub setup_logic_megan {
 
   if ($nl_flags->{'megan'} ) {
     if ( &value_is_true( $nl_flags->{'use_fates'} ) ) {
-       $log->fatal_error("MEGAN can NOT be on when ED is also on.\n" .
+       $log->warning("MEGAN can NOT be on when ED is also on.\n" .
```

Comment:
In looking at this I should update use of ED to FATES.  We hadn't changed use of ED to FATES everywhere when FATES was introduced as the new acronym. There's a few other places for that, and I'll go ahead and do that.

Reply:
I went ahead and completed that.